src/main/java/com/adroll/cantor/HLLCounter.java

```diff
@@ -662,8 +662,15 @@ public static double estimateSize(HLLByteArray Q, double alpha) {
     for(int i = 0; i < Q.length; i++) {
       if(Q.get(i) == 0) {
         count++;
+        E++;
+      } else if (Q.get(i) < 64) {
```

Comment:
I think we might want to make this `< 63` to avoid two's complement issues

I just ran

```
        System.out.println(1L << 31);
        System.out.println(1  << 31);
        System.out.println(1L <<62);
        System.out.println(1L <<63);
```

And got 
```
2147483648
-2147483648
4611686018427387904
-9223372036854775808
```